doc/getting-started.rst

```diff
@@ -486,9 +486,7 @@ versions in their own file:
    >>> write(src, 'buildout.cfg')
 
 Here, we've used the Buildout ``extends`` option to say that
-configurations should be read from the named file (or files) and that
-configuration in the current file should override configuration in the
-extended files.  To continue the example, our ``versions.cfg`` file
+configurations should be read from the named file (or files). These configurations will override and/or extend the current file from which these are loaded (in this case, ``buildout.cfg``).  To continue the example, our ``versions.cfg`` file
```

Comment:
a) Please add linebreaks (to keep within 80 characters per line).

b) The "loading" config file wins over the config files it extends: I believe your sentence says it the wrong way around.

Reply:
>that configuration in the current file should override configuration in the extended files

This states that the included files are overwritten by the file that includes them

>These configurations will override and/or extend the current file from which these are loaded (in this case, ``buildout.cfg``)

States that the included files will overwrite the settings in the file that includes them (so the other way around as above)

So, which one is it ? :D 

I assumed it simply works the way other software uses `include`. And from what I understand, the following holds true for `Buildout`:

```
>>conf_a.conf
    setting_a = True
    include = ./conf_b.conf
>>conf_b.conf
    setting_a = False

>>Resulting config
    setting_a = False
```

Which is what my edit describes. Is this correct? If it isn't, your original sentence is correct.

Reply:
"extends" means that buildout reads that config as a base config. Everything in the current buildout config adds/overrides/changes the extended config.

If you have a base.cfg:

    ....
    [django]
    settings = myproject.developmentsettings

You could have a production.cfg like this:

    [buildout]
    extends = base.cfg

    ....
    [django]
    settings = myproject.productionsettings

So in production.cfg, you'd use myproject.productionsettings.

Reply:
Ah, alright! Thanks for clearing that up. I misunderstood this, then.